src/pages/TaskListTab/TaskListTab.js

```diff
@@ -33,21 +35,19 @@ export class TaskListTab extends Component {
     }
 
     if (key === 'in-progress') {
-      this.state.tasksInWeek[day].forEach(el => {
-        if (el.done === 'in-progress') {
-          task.done = task.done;
-        }
+      if (todayTasks.every((dayTask) => dayTask.done !== 'in-progress')) {
```

Comment:
why do you need this checking? Just make `task.done = false`

Reply:
Because only one task can be in-progress status, so I'm checking all tasks of the current day

Reply:
Ah, I forgot about this requirement. But then, according to spec, no more than two